tool/mustachio/codegen_aot_compiler.dart

```diff
@@ -207,7 +208,8 @@ Future<Method> _redirectingMethod(
 
   var parameters = <Parameter>[];
   for (var context in compiler._usedContextStack) {
-    var contextElement = context.type.element;
+    var contextElement =
+        DartTypeExtension(context.type).element as ClassElement;
```

Comment:
Any cast is a potential land mine.
Are you sure that it is always `ClassElement`, and not `EnumElement` or `MixinElement`?

Reply:
Fixed